### ISSUE TITLE
RTL language layout is incorrect before going into Settings menu

### ISSUE THREAD

**Issue by npark-mozilla** [mozilla-mobile/focus-android#772]
STR:
On English locale android, make sure Focus is killed and not running.
Change to RTL locale from System, and start Focus
* Notice the start of the app takes about 2 seconds, showing grey screen
* Notice the pull-down menu button is on the right side, inside of left side
go to any website, www.google.com
press pull-down menu, and go to Settings
Exit settings
* Notice now the layout shows proper RTL

**Comment by npark-mozilla** [comment]
This was on Nexus 5, 6.0.1.  On above str, the Settings dialog would still not be RTL'ed, just the main browser view.  When following STR is performed, then the settings dialog will be RTL'ed as well.
- Go to Language in Settings menu
- Pick some other language, exit
- Go to Language menu again, pick system language (assuming the system language is already arabic)
- exit

**Comment by npark-mozilla** [comment]
Also noticed that this does not happen on 7.0 simulator, but on 6.0 (simulator) 6.0.1 (device) this happens.

**Comment by pocmo** [comment]
I sometimes saw a temporary wrong layout when switching from LTR to RTL too. This fixed itself when the view was re-displayed and never appeared again.

As long as this is a temporary issue I suggest to:

* Ship v1 as-is. I noticed some related issues in Fennec too. A runtime change (without activity recreation) from LTR to RTL is not really something that "normal" apps do.

* Investigate if their are better option for the next release. I saw that there are some new APIs that didn't exist when the locale switcher was written for Fennec. There might be the opportunity to come up with a better approach.
